core/os/android/adb/commands.go

```diff
@@ -373,6 +373,31 @@ func (b *binding) QueryPerfettoServiceState(ctx context.Context) (*device.Perfet
 	return result, nil
 }
 
+// Currently using Android Q/10, API 29 as ANGLE support cut-off
```

Comment:
How much do we care about capturing on Android 10?

Reply:
We still use a reasonable amount of 10 on the ANGLE team so we care for now, at least. Could potentially update this once 11 lands publicly.